From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] debian/support_out_of_source_tree_build

Modify paths in qtbindings/qtbindingsbase.pri to support
generated_cpp in the builddir. Also output plugins to the
builddir.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

---
 qtbindings/qtbindingsbase.pri |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/qtbindings/qtbindingsbase.pri b/qtbindings/qtbindingsbase.pri
index 8ad8d80..1ead098 100644
--- a/qtbindings/qtbindingsbase.pri
+++ b/qtbindings/qtbindingsbase.pri
@@ -1,8 +1,8 @@
 TEMPLATE = lib
 DEPENDPATH += .
 INCLUDEPATH += .
-DESTDIR = $$PWD/../plugins/script
+DESTDIR = ../../plugins/script
 QT += script
 CONFIG += debug_and_release
-GENERATEDCPP = $$PWD/../generated_cpp
+GENERATEDCPP = ../../generated_cpp
 TARGET=$$qtLibraryTarget($$TARGET)
-- 
tg: (95a3b52..) debian/support_out_of_source_tree_build (depends on: upstream)
